plugins/registry.js

```diff
@@ -167,6 +167,31 @@ export default class PluginRegistry {
         return id;
     }
 
+    // Register a component to render a custom embed preview for post links.
+    // Accepts the following:
+    // - match - A function that receives the embed link and returns a
```

Comment:
It looks like it actually receives an embed object, with various other data in addition to the link. Can you take a stab at documenting what a plugin author can expect here?

Reply:
Sure thing! I just sent a commit adding the embed object keys and a simple description not to tie the documentation it too much to the backend specifics. Let me know if this is close to what you were thinking or if another format or more extended contents are preferred :)